ivadomed/scripts/visualize_and_compare_testing_models.py

```diff
@@ -21,7 +21,7 @@
 
 matplotlib.rcParams['toolbar'] = 'None'  # Remove buttons
 
-gui_env = ['TKAgg', 'GTKAgg', 'Qt4Agg', 'WXAgg']
+gui_env = ['Agg']
```

Comment:
This is just experimental. I'll probably revert this.

But I don't think this loop should be here at all. matplotlib already does this loop: https://matplotlib.org/stable/users/explain/backends.html

> Without a backend explicitly set, Matplotlib automatically detects a usable backend based on what is available on your system and on whether a GUI event loop is already running. The first usable backend in the following list is selected: MacOSX, QtAgg, GTK4Agg, Gtk3Agg, TkAgg, WxAgg, Agg. The last, Agg, is a non-interactive backend that can only write to files.

Reply:
So, I'm pretty sure this whole loop is redundant and can be replaced with just the last line of the block! Because matplotlib already does it. Check this:

```
[kousu@nigiri ~]$ python -c 'import matplotlib; print(matplotlib.get_backend())'
QtAgg
[kousu@nigiri ~]$ DISPLAY="" python -c 'import matplotlib; print(matplotlib.get_backend())'
agg
```

i.e. when I tell it I have no GUI, it automatically falls back to agg.